backend/src/main/java/clf/integra/backend/controller/TestController.java

```diff
@@ -27,4 +37,20 @@ public String test() {
     public String testRepo() {
         return userRepository.getAllUsers().toString();
     }
+
+    //Testing the taxes and fees system
+    @PostMapping("branches/{branchId}/collect-taxes-and-fees")
```

Comment:
Since it starts with branches, this should be going into a `BranchController`, since we are also going to remove this test controller in the near future